### PR TITLE
Create merge_sort_v3.py

### PR DESCRIPTION
**Pull request by maahi07m**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/merge_sort_v3.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Python program for implementation of MergeSort 
+def mergeSort(arr):
```
Comment (cclauss) [inline sorts/merge_sort_v3.py:2]: No type hints or doc tests...  See CONTRIBUTING.md

File: sorts/merge_sort_v3.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+def printList(arr): 
+	for i in range(len(arr)):		 
+		print(arr[i],end=" ") 
+	print() 
+
+# driver code to test the above code 
+if __name__ == '__main__': 
+	arr = [12, 11, 13, 5, 6, 7] 
+	print ("Given array is", end="\n") 
+	printList(arr)
```
Comment (jntushar) [inline sorts/merge_sort_v3.py:44]: You can directly print list using `print(arr)`

Comment (jntushar) [reply sorts/merge_sort_v3.py:44]: instead of creating a separate function

Comment (maahi07m) [reply sorts/merge_sort_v3.py:44]: yup

File: sorts/merge_sort_v3.py (hunk 1 above)
Comment (maahi07m) [reply sorts/merge_sort_v3.py:2]: yeah  i will add that

**Comment by maahi07m** [comment]
> How this algorithm is better than this https://github.com/TheAlgorithms/Python/blob/master/divide_and_conquer/mergesort.py
> 
> if both are same than why to add another file?

yeah! got it

**Comment by maahi07m** [comment]
> Hey @maahi07m,
> Something went wrong with the build.
> 
> TravisCI finished with status `errored`, which means the build failed because of something unrelated to the tests, such as a problem with a dependency or the build process itself.
> 
> [View build log](https://travis-ci.com/TheAlgorithms/Python/builds/172340635)
> 
> ###### TravisBuddy Request Identifier: 62b5fd30-b324-11ea-894e-97d5968798bd

That's because of the intendation error go  check it once or i'll resolve it.

**Pull request closed without merging**